docs/plugin-config.md

```diff
@@ -0,0 +1,76 @@
+# Plugin Configuration
+The `notation-azure-kv` plugin supports plugin configuration to use advanced features.
+
+## Usage
+- Add the plugin configuration when adding the key
+  ```sh
+  notation key add \
+    --plugin azure-kv \
+    --id <key_identifier> \
+    --plugin-config <argument>=<value> \
+    --default
+  ```
+  Then sign the artifact
+  ```sh
+  notation sign <registry>/<repository>@<digest>
+  ```
+
+- Add the plugin config when sign the artifact
+  ```sh
+  notation sign <registry>/<repository>@<digest> \
+    --plugin azure-kv \
+    --id <key_identifier> \
+    --plugin-config <argument>=<value>
+  ```
+
+## Supported arguments
+### ca_certs
+When the `--id` specifies a certificate that has multiple certificates in the certifcate chain, and the intermediate and root certificate were not [merged to Azure Key Vault](https://learn.microsoft.com//azure/key-vault/certificates/create-certificate-signing-request) or the chain was not complete, use `ca_certs` argument to pass the certificate bundle file path to the plugin.
+
+Default: **empty string**
+
+> **Note** Ensure that the certificates in the bundle are correctly ordered: starting from the intermediate certificate that signed the leaf certificate, and ending with the root certificate.
+>
+> They must be concatenated such that each certificate directly validates the preceding one. The following example features two certificates: an intermediate and a root certificate, but your certificate chain may include more or fewer.
+>
+> ```pem
+> -----BEGIN CERTIFICATE-----
+> Base64–encoded intermediate certificate
+> -----END CERTIFICATE-----
+> -----BEGIN CERTIFICATE-----
+> Base64–encoded root certificate
+> -----END CERTIFICATE-----
+> ```
```

Comment:
An end user doesn't always have to manually create a cert bundle on their own. Sometimes, they can download a cert bundle online, for example, https://www.microsoft.com/en-us/download/details.aspx?id=102267. We should mention this kind of cert bundle source in the doc.

Reply:
Updated. The link is Microsoft 365 certificate. I'm not sure whether it is for Microsoft owned product or not. So I change to use the example of digicert.

Reply:
> Updated. The link is Microsoft 365 certificate. I'm not sure whether it is for Microsoft owned product or not. So I change to use the example of digicert.

Yep, I'm just providing a dummy example for your reference.